crates/bevy_ecs/src/spawn.rs

```diff
@@ -133,6 +133,73 @@ impl<R: Relationship, F: FnOnce(&mut RelatedSpawner<R>) + Send + Sync + 'static>
     }
 }
 
+/// A [`SpawnableList`] that adds entities using an iterator of [`Entity`]:
```

Comment:
This isn't a particularly useful description by itself, can you add another line explaining why users might to do this?